### PR TITLE
use the header and footer as a single file

### PR DESCRIPTION
**Pull request by shayanfpg9**
Hi; 👋
when I was opened the webpage for the first time I saw one of the big problems of website is slow loading and when I read the code I undrestand one of the resoans of this problem is copy header and footer codes into the all of the page 😖,
so I think i can fix it and help you to have a better website 🌹

### CODE REVIEW

File: fa/components/header.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+                <li class="nav-item">
+                    <a class="nav-link" data-href="https://github.com/hascal/hascal/tree/main/docs">مستندات</a>
+                </li>
+                
+                <li class="nav-item">
+                    <a class="nav-link" data-href="news.html">اخبار</a>
+                </li>
+
+                <li class="nav-item">
+                    <a class="nav-link" data-href="foundation.html">>بنیاد هاسکال</a>
```
Comment (mmdbalkhi) [inline fa/components/header.html]:
```suggestion
                    <a class="nav-link" data-href="foundation.html">بنیاد هاسکال</a>
```

File: fa/components/header.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+                </li>
+
+                <li class="nav-item">
+                    <a class="nav-link" data-href="style.html">>سبک هاسکال</a>
```
Comment (mmdbalkhi) [inline fa/components/header.html]:
```suggestion
                    <a class="nav-link" data-href="style.html">سبک هاسکال</a>
```

File: fa/components/header.html [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+                </li>
+
+                <li class="nav-item">
+                    <a class="nav-link" data-href="https://github.com/hascal/hascal">>سورس کد</a>
```
Comment (mmdbalkhi) [inline fa/components/header.html]:
```suggestion
                    <a class="nav-link" data-href="https://github.com/hascal/hascal">سورس کد</a>
```

File: fa/index.html [hunk 4]
```diff
@@ -1,93 +1,76 @@
 <!DOCTYPE html>
 <html>
```
Comment (mmdbalkhi) [inline fa/index.html]:
```suggestion
<html lang="fa">
```

File: fa/news.html [hunk 5]
```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
```
Comment (mmdbalkhi) [inline fa/news.html]:
```suggestion
<html lang="fa">
```

**Review by mmdbalkhi — CHANGES_REQUESTED** [review]
Hello @shayanfpg9, We are pleased and glad about your first contribution to Hascal. This is my first round of reviews.
I am not familiar with HTML, and I don't know exactly what the problem is. In the Persian language, Persian `utf-8`characters are not displayed correctly. (on body)

Please Fix this issue.

ps:
@bistcuite do you have any idea?

sc

**Comment by bistcuite** [comment]
@shayanfpg9
Thanks! Do requested changes by @mmdbalkhi to merge this PR to the repository.

**Comment by shayanfpg9** [comment]
I'm so sorry for the trouble but when I tested it on my system everything worked fine but I added your suggestions and now I think it should work for you too 🌹

**Comment by mmdbalkhi** [comment]
Thank you so much @shayanfpg9 for working on this.

**Pull request merged**